docs/collecting-data/collecting-from-own-applications/react-native-tracker/anonymous-tracking/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Anonymous Tracking"
+date: "2022-08-30"
+sidebar_position: 25
+---
+
+# Anonymous Tracking
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+:::info
+
+This feature is available since v1.3.
+
+:::
+
+Anonymous tracking is a tracker feature that enables anonymising various user and session identifiers to support user privacy in case consent for tracking the identifiers is not given.
```

Comment:
This preamble is a copy of https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/mobile-trackers/anonymous-tracking/, as far as I can tell. Can we put it inside a reusable fragment? (In both places, naturally!) See the README for instructions.

Reply:
Extracted into a reusable fragment. Thanks for the comments!